libs/ledger-live-common/src/config/quotesRate.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Default to 20 seconds for quotes refresh rate
+ */
+export const ratesExpirationThreshold = 20 * 1000;
```

Comment:
I'm not clear on why we're using a function when what we're defining is a constant. Creating tests for a constant seems a bit overengineered to me. Moreover, this constant is entirely specific to "swap." Wouldn't it make more sense to rename the file "timeout.ts" to "time" and use it like this?
```
ledger-live/libs/ledger-live-common/src/exchange/swap/const/time.ts

export const DEFAULT_SWAP_TIMEOUT_MS = 10000;
export const SWAP_RATE_INTERVAL = 10000;